containers/runtime/xds/config/README.md

```diff
@@ -0,0 +1,37 @@
+
+The configuration files should follow the structure in the daufult_config.json file.
+
+* Resources: An array holding the resouces, there are currently 8 types of resources, the index of this array corresponds to the resource type.
+* Version: Versions indicate the version of current group of resource. This field is served as key in VersionMap.
+* TTL: Optional fields to set TTL for each resource item.
+* Items: Items are maps within each resource type, the maps' keys are the individual resource names and the values are the actual resources with an optional TTL
+* VersionMap: VersionMap holds the current hash map of all resources in the snapshot. This field should remain nil until it is used. In our use case, we only unmarshal the configuration into Snapshot, the VersionMap remains nil. VersionMap is only to be used with delta xDS.
+
+The source of truth for the fields within each resource in the resource map are their protos.
+The 8 types of resources:
+
+* Listener: [config.listener.v3.Listener proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/config/listener/v3/listener.proto#L39)
+* Endpoint: [config.endpoint.v3.ClusterLoadAssignment proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/config/endpoint/v3/endpoint.proto#L33)
+* Cluster: [config.cluster.v3.Cluster proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/config/cluster/v3/cluster.proto#L47)
+* Route: [config.route.v3.RouteConfiguration proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/config/route/v3/route.proto#L26)
+* ScopedRoute: [config.route.v3.ScopedRouteConfiguration proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/config/route/v3/scoped_route.proto#L83)
+* ExtensionConfig: [config.core.v3.ExtensionConfigSource proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/config/core/v3/extension.proto#L47)
+* Secret: [extensions.transport_sockets.tls.v3.Secret proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/extensions/transport_sockets/tls/v3/secret.proto#L40)
+* Runtime: [service.runtime.v3.Runtime proto](https://github.com/envoyproxy/envoy/blob/3865cbaec08c5ce44b439eb77e46fe866e37e81a/api/envoy/service/runtime/v3/rtds.proto#L50)
+
+## Default configuration JSON file
+
+The daufult_config.json file contains a piece of configuration that have two listeners with name: `default_testEnvoyListenerName` and `default_testGrpcListenerName`, one Cluster resource, one Route resource and one Endpoint resource. The two listeners are pointing to the same Cluster, eventually the same Endpoint resource.
```

Comment:
`default_config.json`. have -> has, "with name:" -> named. I wonder if it is necessary to capitalize 'Cluster', 'Route' etc and to say 'resource' every time, or just lowercase and without 'resource' everywhere would be clearer.

Reply:
I think what you recommend works. Updated